src/docs/data/builders/tree.ts

```diff
@@ -0,0 +1,183 @@
+import { DESCRIPTIONS, KBD, PROPS } from '$docs/constants';
+import type { KeyboardSchema } from '$docs/types';
+import { builderSchema, elementSchema } from '$docs/utils';
+import type { BuilderData } from '.';
+
+const BUILDER_NAME = 'tree-view';
+
+const builder = builderSchema(BUILDER_NAME, {
+	title: 'createTreeViewBuilder',
+	description: DESCRIPTIONS.BUILDER('tree-view'),
+	props: [
+		PROPS.FORCE_VISIBLE
+	],
+	elements: [
+		{
+			name: 'tree',
+			description: 'The builder store used to create the tree.',
+			link: '#tree',
+		},
+		{
+			name: 'label',
+			description: 'The builder store used to create the label.',
+			link: '#label',
+		},
+		{
+			name: 'item',
+			description: 'The builder store used to create a tree item.',
+			link: '#item',
+		},
+		{
+			name: 'group',
+			description: 'The builder store used to create a tree group.',
+		}
+	],
+	states: [
+		{
+			name: 'collapsedItems',
+			description: 'The list of IDs of items, who have a group that is not expanded.'
+		}
+	],
+	helpers: [
+		{
+			name: 'focusedItem',
+			description: 'The list item that is currently focused.'
+		},
+		{
+			name: 'selectedItem',
+			description: 'The list item that is currently selected.'
+		}
+	]
+});
+
+const tree = elementSchema('tree', {
+	title: 'tree',
+	description: 'The tree <ul> element.',
+	dataAttributes: [
+		{
+			name: 'role',
+			value: 'tree'
+		},
+		{
+			name: 'aria-labelledby',
+			value: 'The unique ID assigned to the label element.'
+		},
+		{
+			name: 'data-melt-id',
+			value: 'A unique ID.'
+		}
+	]
+});
+
+const label = elementSchema('label', {
+	title: 'label',
+	description: 'The label element.',
+	dataAttributes: [
+		{
+			name: 'id',
```

Comment:
It should be data-id. If it's actually setting an id, we currently dont display that on the docs

Reply:
Hey @TGlide, I think we would need the id here for the `aria-labelldby` attribute. We could also not add it through the builder and just show it in the preview code. How would I document that the id gets added through the builder though?

![image](https://github.com/melt-ui/melt-ui/assets/20195536/0b0fb251-b388-469b-97c2-11a1c4377175)

Right now I just added it in the data attributes. In the element schema I don't see another option of where to put it.